common/metrics-core/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Re-export counter types from prometheus crate
+pub use prometheus::{
+    gather, register_histogram, register_histogram_vec, register_int_counter,
+    register_int_counter_vec, register_int_gauge, register_int_gauge_vec, Encoder, Histogram,
+    HistogramVec, IntCounter, IntCounterVec, IntGauge, IntGaugeVec, TextEncoder,
+};
```

Comment:
If this is all the crate does, then why don't we just have folks use prometheus directly instead of adding an extra crate which just exports prometheus types?

Reply:
Look at the links... Without this, it appears we're losing metrics.

Reply:
Something else must be going on, because this is literally just re-exporting types from the prometheus crate instead of using prometheus directly. I'm not saying we can't do this, just saying that doing this shouldn't "fix" lost metrics

Reply:
Some context: I originally had reexported prometheus because all the calling code imported both libra-metrics and prometheus. So I reexported the needed prometheus types from libra-metrics to make it easier to use and remove the need for all callers to declare both dependencies directly.

Reply:
I agree, there is definitely a quirk here. At the same time, this definitely fixes the quirk albeit maybe not in the sexiest of fashions. I'm willing to make our metrics repo depend on push-metrics, is that agreeable?